kyuubi-server/src/main/scala/org/apache/kyuubi/parser/KyuubiParser.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.parser
+
+import scala.util.control.NonFatal
+
+import org.antlr.v4.runtime.{CharStream, CharStreams, CodePointCharStream, CommonTokenStream, IntStream}
+import org.antlr.v4.runtime.atn.PredictionMode
+import org.antlr.v4.runtime.misc.{Interval, ParseCancellationException}
+
+import org.apache.kyuubi.parser.plan.KyuubiNode
+import org.apache.kyuubi.sql.{KyuubiSqlBaseLexer, KyuubiSqlBaseParser, KyuubiSqlBaseParserBaseListener}
+
+class KyuubiParser {
+
+  val astBuilder = new KyuubiAstBuilder
+
+  def parsePlan(sqlText: String): KyuubiNode = parse(sqlText) { parser =>
+    astBuilder.visit(parser.singleStatement) match {
+      case plan: KyuubiNode => plan
+      case _ =>
+        throw new ParseException(s"Unsupported SQL statement $sqlText")
```

Comment:
if we will pass through, why do we have unsupported statements?

Reply:
yes, can remove, any sql will pass through(at least)